### PR TITLE
[PR1] DEV-12: MVP-1 logger

### PR DESCRIPTION
**Pull request by lipezaballa**
It creates a file that overwrites the previous run, from each package it prints the props we are interested in, as well as each measurement.

### CODE REVIEW

File: src/DataTransfer/application/data_transfer.go [hunk 1]
```diff
@@ -20,13 +19,20 @@ func New(rawBoards map[string]excelParser.Board) DataTransfer {
...
+func (dataTransfer DataTransfer) Invoke(getPacketUpdate func() packetParser.PacketUpdate, logFile *os.File) {
+
 	for {
 		update := getPacketUpdate()
 		dataTransfer.data.UpdatePacket(update)
 		packet := dataTransfer.data.GetPacket(update.ID)
 
-		spew.Dump(packet)
-		<-time.After(time.Second)
+		titlePacket := fmt.Sprintf(`Id: %v    Name: %v    Count: %v    CycleTime: %v`,
```
Comment (smorsua) [inline src/DataTransfer/application/data_transfer.go:29]: Mete lo de escribir en el file dentro de una función.

File: src/DataTransfer/application/data_transfer.go [hunk 2]
```diff
@@ -20,13 +19,20 @@ func New(rawBoards map[string]excelParser.Board) DataTransfer {
 	}
 }
 
-func (dataTransfer DataTransfer) Invoke(getPacketUpdate func() packetParser.PacketUpdate) {
+func (dataTransfer DataTransfer) Invoke(getPacketUpdate func() packetParser.PacketUpdate, logFile *os.File) {
+
```
Comment (smorsua) [inline src/DataTransfer/application/data_transfer.go:23]:
```suggestion
 defer logFile.Close()
```

**Review by smorsua — CHANGES_REQUESTED** [review]
LGTM, I'll aprove it once you make the changes suggested

**Review by jmaralo — DISMISSED** [review]
looks great, we'll need to perform benchmarks to make sure everything is fast enough

**Review by jmaralo — DISMISSED** [review]

**Review by smorsua — APPROVED** [review]
Buen trabajo

**Review by smorsua — APPROVED** [review]

**Review by smorsua — DISMISSED** [review]

**Review by smorsua — APPROVED** [review]

**Review by jmaralo — APPROVED** [review]

**Pull request merged**